cmd/grype/cli/options/database.go

```diff
@@ -41,6 +42,7 @@ func DefaultDatabase(id clio.Identification) Database {
 		ValidateAge: true,
 		// After this period (5 days) the db data is considered stale
 		MaxAllowedBuiltAge:     defaultMaxDBAge,
+		RequireUpdateCheck:     true,
```

Comment:
If we default this to `false`, this PR as written would essentially have the same behavior as grype today with what I believe is a useful change that `grype db update` will fail if it is unable to determine if the db is up to date because the listing file failed.